com.unity.netcode.gameobjects/Tests/Runtime/Serialization/NetworkObjectReferenceTests.cs

```diff
@@ -25,9 +25,12 @@ private class TestNetworkBehaviour : NetworkBehaviour
 
             public GameObject RpcReceivedGameObject;
 
+            public bool ReceivedRpc;
```

Comment:
Your test passes without this variable - should it be in this PR at all?

Reply:
No, it shouldn't. I added this in a previous iteration of the test, then changed directions and forgot to revert this.